Bluetooth CI cannot pass twister tests of hci_usb sample

**Describe the bug**
Bluetooth CI cannot pass twister test steps. It cannot build hci_usb sample.

Seems broken code was introduced in this PR: https://github.com/zephyrproject-rtos/zephyr/pull/70417

**To Reproduce**
It is reproduced in my PRs CI step. Here: https://github.com/zephyrproject-rtos/zephyr/actions/runs/8419095196/job/23051161569?pr=70437

**Expected behavior**
CI steps passed

**Impact**
blocks PR merge

**Logs and console output**
```
CMakeFiles/app.dir/src/main.c.obj.d -o CMakeFiles/app.dir/src/main.c.obj -c /__w/zephyr/zephyr/samples/bluetooth/hci_usb/src/main.c
/__w/zephyr/zephyr/samples/bluetooth/hci_usb/src/main.c: In function 'enable_usb_device_next':
/__w/zephyr/zephyr/samples/bluetooth/hci_usb/src/main.c:17:43: error: too few arguments to function 'sample_usbd_init_device'
   17 |         struct usbd_contex *sample_usbd = sample_usbd_init_device();
      |                                           ^~~~~~~~~~~~~~~~~~~~~~~
In file included from /__w/zephyr/zephyr/samples/bluetooth/hci_usb/src/main.c:13:
/__w/zephyr/zephyr/samples/subsys/usb/common/sample_usbd.h:30:21: note: declared here
   30 | struct usbd_contex *sample_usbd_init_device(usbd_msg_cb_t msg_cb);
      |                     ^~~~~~~~~~~~~~~~~~~~~~~
ninja: build stopped: subcommand failed.
```

--- a/samples/bluetooth/hci_usb/src/main.c
+++ b/samples/bluetooth/hci_usb/src/main.c
@@ -14,7 +14,7 @@
 
 static int enable_usb_device_next(void)
 {
-	struct usbd_contex *sample_usbd = sample_usbd_init_device();
+	struct usbd_contex *sample_usbd = sample_usbd_init_device(NULL);
 
 	if (sample_usbd == NULL) {
 		printk("Failed to initialize USB device");

